core/src/io.jl

```diff
@@ -125,16 +125,27 @@ function write_basin_output(model::Model)
     tsteps = datetime_since.(timesteps(model), config.starttime)
     ntsteps = length(tsteps)
 
-    time = convert.(Arrow.DATETIME, repeat(tsteps; inner = nbasin))
-    node_id = repeat(basin_id; outer = ntsteps)
-
     storage = reshape(vec(sol), nbasin, ntsteps)
     level = zero(storage)
     for (i, basin_storage) in enumerate(eachrow(storage))
         level[i, :] =
             [get_area_and_level(p.basin, i, storage)[2] for storage in basin_storage]
     end
 
+    return storage, level, tsteps, ntsteps
```

Comment:
Shall we remove `ntsteps` from this function, and just calculate it from tsteps in the other function?
Also I think I'd prefer to return a single NamedTuple of Vectors here. That is also a valid Tables.jl table type.

`(; time=tsteps, storage, level)`